training/v5p/DeepSeek3-671B-MaxText/README.md

```diff
@@ -0,0 +1,61 @@
+# Instructions for training DeepSeek-671B-MaxText on TPU v5p-1024
+
+## XPK setup
+Please follow this [link](https://github.com/gclouduniverse/reproducibility/tree/main/Training/TPU-v5p/XPK_README.md) to create your GKE cluster with XPK
+
+## Prep for Maxtext
+
+1. Clone [Maxtext](https://github.com/AI-Hypercomputer/maxtext) repo
+```
+git clone https://github.com/AI-Hypercomputer/maxtext.git
+cd maxtext
+git checkout 3eb77db3c94580f56f1b738f8d254b03bd205e35
+```
+
+2. Run the following commands to build the docker image
+```
+bash docker_build_dependency_image.sh DEVICE=tpu MODE=stable JAX_VERSION=0.7.0
+```
```

Comment:
Do we want to do something like https://github.com/AI-Hypercomputer/maxtext/pull/1905 to pin the dependency set?